lib/rust/parser/engine/src/lib.rs

```diff
@@ -0,0 +1,19 @@
+#![feature(test)]
+#![deny(unconditional_recursion)]
+#![warn(missing_copy_implementations)]
+#![warn(missing_debug_implementations)]
+#![warn(missing_docs)]
+#![warn(trivial_casts)]
+#![warn(trivial_numeric_casts)]
+#![warn(unsafe_code)]
+#![warn(unused_import_braces)]
```

Comment:
Why?

Reply:
Don't we use these warnings in every crate?

Reply:
Oh, sorry. I misread this as `allow`.